ArchiSteamFarm/Core/Statistics.cs

```diff
@@ -108,7 +107,7 @@ internal sealed class Statistics : IAsyncDisposable {
 		}
 
 		try {
-			Uri request = new($"{URL}/Api/HeartBeat");
+			Uri request = new(ASFServer.URL, "/Api/HeartBeat");
```

Comment:
I would suggest moving all the requests to API server to `ASFServer` class, which should be more convenient than searching for certain requests in the code of there classes (e.g. in case you will need to change API)

Reply:
Good point, statistics logic stays in the `Statistics.cs`, but API endpoints returning data can be there.